package.json

```diff
@@ -121,12 +122,13 @@
     "rollup-plugin-node-resolve": "^5.0.1",
     "rollup-plugin-svelte": "^5.0.3",
     "serialize-javascript": ">=3.1.0",
+    "standard-version": "^9.0.0",
     "svelte": "^3.18.2",
     "tarima": "^4.8.5",
-    "testcafe": "^1.9.4"
+    "testcafe": "^15.1.317922"
```

Comment:
I'm afraid of this change, please revert.

Reply:
Ok